### PR TITLE
boards/modalai_fc-v1 Maintenance Updates for v1.11 beta

### PR DESCRIPTION
**Pull request by modaltb**
**Describe problem solved by this pull request**
Small maintenance update for modalai_fc-v1 board support
- Fixes inverted PWM ordering introduced by 72c742f
- Fixes arming issue caused by integration of e9890d0
- [backport] update to single wire from 72c742f
- threw in the RM3100 into sensor startup

**Test data / coverage**
Indoor flight test was good, outdoor flight testing scheduled early next week

### CODE REVIEW

**Comment by modaltb** [comment]
The CI builds look OK?  Failing on archiving of artifacts

**Comment by dagar** [comment]
> The CI builds look OK? Failing on archiving of artifacts

Jenkins is a bit of a mess right now (running out of space) and I'm hoping to migrate all the compile jobs to Github actions.

File: boards/modalai/fc-v1/src/board_config.h [hunk 1]
```diff
@@ -277,8 +277,8 @@
 
 /* Power supply control and monitoring GPIOs */
 
-#define BOARD_NUMBER_BRICKS             0
-#define BOARD_NUMBER_DIGITAL_BRICKS     0
+#define BOARD_NUMBER_BRICKS             (1)
```
Comment (dagar) [inline boards/modalai/fc-v1/src/board_config.h]: Could you explain this one?

Comment (modaltb) [reply boards/modalai/fc-v1/src/board_config.h]: Perhaps a hack but in order to get the vehicle to arm...
https://github.com/PX4/Firmware/blob/4698a09b987428b5d26928935e224a47b827e77e/src/modules/commander/Commander.cpp#L1401
https://github.com/PX4/Firmware/blob/4698a09b987428b5d26928935e224a47b827e77e/src/drivers/adc/ADC.cpp#L218

Comment (modaltb) [reply boards/modalai/fc-v1/src/board_config.h]: Checking this now to see if there's a better way to do this.

**Comment by modaltb** [comment]
OK, I think this is more correct.  This board doesn't have battery monitoring through ADC or GPIO, we interface through an I2C bus to a power module and get status that way.  So I believe not defining number of bricks is the way to go such that the board_config.h is accurate to the hardware, and we're leaving BOARD_ADC_BRICK_VALID set to 1 to have the ADC checking happy.

**Comment by modaltb** [comment]
Hi Daniel, let me know if you have any issues with above and I can update, thanks!

File: boards/modalai/fc-v1/src/board_config.h [hunk 2]
```diff
@@ -309,12 +304,8 @@
 #define HRT_PPM_CHANNEL         /* T8C1 */  1  /* use capture/compare channel 1 */
 #define GPIO_PPM_IN             /* PI5 T8C1 */ GPIO_TIM8_CH1IN_2
 
-#define RC_UXART_BASE                      STM32_USART6_BASE
 #define RC_SERIAL_PORT                     "/dev/ttyS5"
-#define BOARD_HAS_SINGLE_WIRE              1 /* HW is capable of Single Wire */
-#define BOARD_HAS_SINGLE_WIRE_ON_TX        0 /* HW default is wired as Single Wire On RX pin */
-#define BOARD_HAS_RX_TX_SWAP               1 /* HW Can swap TX and RX */
-#define RC_SERIAL_PORT_IS_SWAPED           1 /* Board wired with RC's TX is on cpu RX */
+#define BOARD_HAS_SINGLE_WIRE
```
Comment (dagar) [inline boards/modalai/fc-v1/src/board_config.h]: I think this is obsolete?

Comment (modaltb) [reply boards/modalai/fc-v1/src/board_config.h]: damn, I thought I updated to correct usage, let me fix.

**Comment by modaltb** [comment]
OK sorry about those two brain farts on this PR..... brain was elsewhere apparently...

**Pull request merged**